src/components/package/show/package-show.js

```diff
@@ -517,18 +481,12 @@ class PackageShow extends Component {
           }
         </Accordion>
 
-        <Accordion
+        <AgreementsSection
```

Comment:
The component is called Accordion section, but it is accordion. please rename to AgreementsAccordion